lib/author.rb

```diff
@@ -0,0 +1,30 @@
+require './lib/book'
+class Author 
+  attr_reader :books
+
+  def initialize(name_hash)
+    @name = {
+      first_name: name_hash[:first_name],
+      last_name: name_hash[:last_name]
+    }
```

Comment:
this is taking a hash as input, and stores it in another hash... why not just store it as `@name_hash=name_hash`